PubsubTableProviderIT.testSQLSelectsArrayAttributes[0] failing in SQL PostCommit

Failing since https://ci-beam.apache.org/job/beam_PostCommit_SQL/6260/

```

java.lang.AssertionError: Did not receive signal on projects/apache-beam-testing/subscriptions/result-subscription--1731534761628855545
in 60s
	at org.apache.beam.sdk.io.gcp.pubsub.TestPubsubSignal.pollForResultForDuration(TestPubsubSignal.java:341)
	at
org.apache.beam.sdk.io.gcp.pubsub.TestPubsubSignal.waitForSuccess(TestPubsubSignal.java:288)
	at org.apache.beam.sdk.extensions.sql.meta.provider.pubsub.PubsubTableProviderIT.testSQLSelectsArrayAttributes(PubsubTableProviderIT.java:256)

```


Imported from Jira [BEAM-12320](https://issues.apache.org/jira/browse/BEAM-12320). Original Jira may contain additional context.
Reported by: bhulette.

--- a/sdks/java/extensions/sql/src/test/java/org/apache/beam/sdk/extensions/sql/meta/provider/pubsub/PubsubTableProviderIT.java
+++ b/sdks/java/extensions/sql/src/test/java/org/apache/beam/sdk/extensions/sql/meta/provider/pubsub/PubsubTableProviderIT.java
@@ -78,7 +78,6 @@ import org.apache.beam.vendor.calcite.v1_28_0.org.apache.calcite.jdbc.CalciteCon
 import org.hamcrest.Matcher;
 import org.joda.time.Duration;
 import org.joda.time.Instant;
-import org.junit.Ignore;
 import org.junit.Rule;
 import org.junit.Test;
 import org.junit.runner.RunWith;
@@ -196,7 +195,6 @@ public class PubsubTableProviderIT implements Serializable {
     resultSignal.waitForSuccess(timeout);
   }
 
-  @Ignore("https://github.com/apache/beam/issues/20937")
   @Test
   public void testSQLSelectsArrayAttributes() throws Exception {
 

